src/main.rs

```diff
@@ -176,6 +176,7 @@ async fn run(args: Args) -> anyhow::Result<()> {
                         logs_collector_id,
                         args.data_dir,
                         args.ping_interval,
+                        args.ping_interval,
```

Comment:
I would make it a separate parameter. It should probably be 60 secs by default

Reply:
done